code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -108,7 +108,7 @@
 		force = 30
 		w_class = W_CLASS_LARGE
 		sharpness = sharpness_on
-		sharpness_flags = initial(sharpness_flags)
+		sharpness_flags = SHARP_TIP | SHARP_BLADE | INSULATED_EDGE | HOT_EDGE | CHOPWOOD
```

Comment:
That's because its initial flags are probably blank buddy.

Currently the code sets its sharpness in New()